src/matlab2tikz.m

```diff
@@ -1911,6 +1919,11 @@ function assertRegularAxes(handle)
             || ~abs(lineWidth-matlabDefaultLineWidth) <= m2t.tol
         lineOpts = opts_add(lineOpts, 'line width', sprintf('%.1fpt', lineWidth));
     end
+
+    % print no lines
+    if isNone(lineStyle) || lineWidth==0
```

Comment:
Indeed we do have such a function.
However, that is for `line` objects (with markers, deviations, ...), while this relates to the lines of an `axes` object. Probably we could use that function (it should be robust enough to handle it), I'm just not sure what the best is:
 - slightly abuse that function for more readable code, or
 - do explicit checks for slightly more complex code